src/core/Akka.Persistence/Journal/AsyncWriteJournal.cs

```diff
@@ -217,8 +219,8 @@ async Task ProcessDelete()
             {
                 try
                 {
-                    await _breaker.WithCircuitBreaker((message, awj: this), state =>
-                        state.awj.DeleteMessagesToAsync(state.message.PersistenceId, state.message.ToSequenceNr));
+                    await _breaker.WithCircuitBreaker((message, awj: this), (state, ct) =>
+                        state.awj.DeleteMessagesToAsync(state.message.PersistenceId, state.message.ToSequenceNr, ct));
```

Comment:
Example of how `CircuitBreaker` will pass in its `CancellationToken` to the method it is protecting as a cancellation signal.